.github/workflows/release.yml

```diff
@@ -0,0 +1,13 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'develop'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
I don't understand why only `checkout` is the only step here. Nothing happens after checking out the code?

Reply:
It's just to enable the GH workflow. It will change with final PR.

Reply:
Ok. Why not include all the work in one PR and then review all of it? Since it's one ticket?

Reply:
As I need to test workflow events with correct branches and test it with dispatch event while testing the workflow. Merging it to default branch makes it easy to clone it PR branches. Also this PR won't make any impact to existing environments.